test/cpp/tensorexpr/test_simplify.cpp

```diff
@@ -3950,5 +3950,19 @@ void testSimplifySyncThreads() {
   }
 }
 
+void testSimplifyRampSubBroadcast() {
+  KernelScope kernel_scope;
+  int num_lanes = 4;
+  ExprHandle ramp = Ramp::make(ExprHandle(0), ExprHandle(6), num_lanes);
+  ExprHandle broadcast = Broadcast::make(ExprHandle(-5), num_lanes);
+  ExprHandle simplified = IRSimplifier::simplify(ramp - broadcast);
+  Ramp* newRamp = simplified.AsNode<Ramp>();
+  ASSERT_NE(dynamic_cast<const IntImm*>(newRamp->base()), nullptr);
```

Comment:
Nit: This would be a little clearer using the macros in this file, e.g.:
```
IS_NODE_WITH_NAME(IntImm, newRamp->base(), base);
ASSERT_EQ(base->Value(), 5);
```

Reply:
Done, thanks.